src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/infrastructure/client/HomeOfficeSendNotificationClient.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.hmcts.reform.iahomeofficeintegrationapi.infrastructure.client;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientResponseException;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.AsylumCaseDefinition;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.infrastructure.client.util.HomeOfficeApiUtil;
+
+@Service
+public class HomeOfficeSendNotificationClient {
+
+    private final RestTemplate restTemplate;
+    private final String homeOfficeSendInstructUrl;
+    private HomeOfficeApiUtil homeOfficeApiUtil;
+
+    public HomeOfficeSendNotificationClient(
+        @Value("${homeoffice.endpoint}") String homeOfficeEndpoint,
+        @Value("${homeoffice.instruct.uri}") String homeOfficeSendInstructUrl,
+        RestTemplate restTemplate,
+        HomeOfficeApiUtil homeOfficeApiUtil
+    ) {
+        this.restTemplate = restTemplate;
+        this.homeOfficeSendInstructUrl = homeOfficeEndpoint + homeOfficeSendInstructUrl;
+        this.homeOfficeApiUtil = homeOfficeApiUtil;
+    }
+
+    public int sendNotification(AsylumCase asylumCase) {
+
+        try {
+            ResponseEntity<String> response = restTemplate.postForEntity(
+                homeOfficeSendInstructUrl,
+                createRequest(asylumCase),
+                String.class);
+
+            return response.getStatusCode().value();
+
+        } catch (RestClientResponseException clientEx) {
+            throw new HomeOfficeResponseException(
+                clientEx.getResponseBodyAsString(),
+                clientEx
+            );
+
+        }
+    }
+
+    public HttpEntity<Map<String, Object>> createRequest(AsylumCase asylumCase) {
+        Map<String, Object> body = new HashMap<>();
```

Comment:
`Map<String, Object>` is just ok to send request body, you could add request model with named fields, that will be more clear. Please take a look at client examples below:
https://github.com/hmcts/ia-timed-event-service/tree/master/src/main/java/uk/gov/hmcts/reform/timedevent/infrastructure/clients
or here:
https://github.com/hmcts/ia-case-payments-api/tree/master/src/main/java/uk/gov/hmcts/reform/iacasepaymentsapi/infrastructure/clients
It is good to use Feign library, it takes care about parsing and many more things. Lombok usage is optional only where you are confident with annotation

Reply:
Mapped to model
https://github.com/hmcts/ia-home-office-integration-api/pull/75